File: app/src/main/java/com/aleksandra/go4mytrip/TripModel.java
package com.aleksandra.go4mytrip;


public class TripModel {
    private String TripId;
    private String Title;
    private String NamePlace;
    private String Coordinate;
    private int ImageTrip;
    private String TripDate;
    private String TripDateEnd;
    private String TimeStart;
    private String TimeEnd;


    public TripModel() {

    }

    public TripModel(String id, String title, String namePlace, String coordinate, int imageTrip, String tripDate, String tripDateEnd, String timeStart, String timeEnd) {
        TripId = id;
        Title = title;
        NamePlace = namePlace;
        Coordinate = coordinate;
        ImageTrip = imageTrip;
        TripDate = tripDate;
        TripDateEnd = tripDateEnd;
        TimeStart = timeStart;
        TimeEnd = timeEnd;

    }

    public String getTimeStart() {
        return TimeStart;
    }

    public void setTimeStart(String timeStart) {
        TimeStart = timeStart;
    }

    public String getTimeEnd() {
        return TimeEnd;
    }

    public void setTimeEnd(String timeEnd) {
        TimeEnd = timeEnd;
    }

    public String getTripId() {
        return TripId;
    }

    public void setTripId(String tripId) {
        TripId = tripId;
    }

    public String getNamePlace() {
        return NamePlace;
    }

    public void setNamePlace(String namePlace) {
        NamePlace = namePlace;
    }

    public String getCoordinate() {
        return Coordinate;
    }

    public void setCoordinate(String coordinate) {
        Coordinate = coordinate;
    }

    public String getTripDateEnd() {
        return TripDateEnd;
    }

    public void setTripDateEnd(String tripDateEnd) {
        TripDateEnd = tripDateEnd;
    }

    public String getTripDate() {
        return TripDate;
    }

    public void setTripDate(String tripDate) {
        TripDate = tripDate;
    }

    public void setImageTrip(int imageTrip) {
        ImageTrip = imageTrip;
    }

    public int getImageTrip() {
        return ImageTrip;
    }

    public String getTitle() {
        return Title;
    }

    public void setTitle(String title) {
        Title = title;
    }


}

File: app/src/main/java/com/aleksandra/go4mytrip/TripModel.kt
package com.aleksandra.go4mytrip

import android.os.Parcel
import android.os.Parcelable

data class TripModel (var tripId: String? = null, var title: String? = null, var namePlace: String? = null, var coordinate: String? = null, var imageTrip:Int = 0, var tripDate: String? = null,
                                 var tripDateEnd: String? = null, var timeStart: String? = null, var timeEnd: String? = null) : Parcelable {
    constructor(parcel: Parcel) : this(
            parcel.readString(),
            parcel.readString(),
            parcel.readString(),
            parcel.readString(),
            parcel.readInt(),
            parcel.readString(),
            parcel.readString(),
            parcel.readString(),
            parcel.readString())

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeString(tripId)
        parcel.writeString(title)
        parcel.writeString(namePlace)
        parcel.writeString(coordinate)
        parcel.writeInt(imageTrip)
        parcel.writeString(tripDate)
        parcel.writeString(tripDateEnd)
        parcel.writeString(timeStart)
        parcel.writeString(timeEnd)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<TripModel> {
        override fun createFromParcel(parcel: Parcel): TripModel {
            return TripModel(parcel)
        }

        override fun newArray(size: Int): Array<TripModel?> {
            return arrayOfNulls(size)
        }
    }
}
